src/app/routes/enrolment/view-requests/components/expiration-date/expiration-date.component.ts

```diff
@@ -18,6 +18,7 @@ import { takeUntil } from 'rxjs/operators';
 export class ExpirationDateComponent implements OnInit, OnDestroy {
   @Input() defaultValidityPeriod: number;
   @Output() add: EventEmitter<number> = new EventEmitter<number>();
+  @Output() remove: EventEmitter<number> = new EventEmitter<number>();
```

Comment:
```suggestion
  @Output() remove: EventEmitter<void> = new EventEmitter<void>();
```
This way you won't accidentally emit value.

Reply:
Ah good catch, I forgot to update! Thanks